content/transactions/setregularkey.md

```diff
@@ -2,7 +2,15 @@
 
 [[Source]<br>](https://github.com/ripple/rippled/blob/4239880acb5e559446d2067f00dabb31cf102a23/src/ripple/app/transactors/SetRegularKey.cpp "Source")
 
-A SetRegularKey transaction changes the regular key associated with an address.
+A SetRegularKey transaction assigns, changes, or removes the regular key associated with an account.
+
+For more information about regular and master keys, see [Understanding Master and Regular Keys](reference-transaction-format.html#understanding-master-and-regular-keys).
+
+You can protect your master secret by assigning regular keys to an account and using them instead of the master keys to sign transactions whenever possible. If your regular keys are compromised, but the master keys are not, you can use a SetRegularKey transaction to regain control of your account.
```

Comment:
Nit: doing this doesn't technically protect your master secret. It protects your account because you can still use your master secret _if_ you've been responsible about keeping your master secret... secret.